packages/test-exclude/index.js

```diff
@@ -57,7 +57,7 @@ class TestExclude {
             this.excludeNodeModules &&
             !this.exclude.includes('**/node_modules/**')
         ) {
-            this.exclude.push('**/node_modules/**');
+            this.exclude = this.exclude.concat('**/node_modules/**');
```

Comment:
This may appear unrelated but now `this.exclude === exportFunc.defaultExclude` which does not contain `**/node_modules/**` so without this shallow copy we would append the default list (this was caught by the tests).  Previously we were appending any `opts.exclude` that did not include `**/node_modules/**`, though that was unlikely to cause any real problem.